appservice/src/createAppService/IAppServiceWizardContext.ts

```diff
@@ -72,4 +73,16 @@ export interface IAppServiceWizardContext extends IResourceGroupWizardContext, I
      */
 
     recommendedSiteRuntime?: LinuxRuntimes[];
+
+    /**
+     * Application insights components are necessary for Function apps log streaming.  By default, we should instance
+     * one for the user if there is a data farm available within the same region as the web app
+     */
+    applicationInsights?: ApplicationInsightsComponent;
```

Comment:
How about you consistently use `appInsights` instead of `applicationIsights` (similar to what you did with `AppInsightsCreateStep`)

Also feels like this one should be named `appInsightsComponent`. Technically we could add `component` to all the names, but that seems like it would get excessive. I guess this is the only one that bothers me the most